diff --git a/source/Files.cpp b/source/Files.cpp
index 1578073e8..4d33053e5 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -117,7 +117,7 @@ void Files::Init(const char * const *argv)
 	// Special case, for Linux: the resource files are not in the same place as
 	// the executable, but are under the same prefix (/usr or /usr/local).
 	static const string LOCAL_PATH = "/usr/local/";
-	static const string STANDARD_PATH = "/usr/";
+	static const string STANDARD_PATH = "/app/";
 	static const string RESOURCE_PATH = "share/games/endless-sky/";
 	if(!resources.compare(0, LOCAL_PATH.length(), LOCAL_PATH))
 		resources = LOCAL_PATH + RESOURCE_PATH;
