diff --git a/source/Files.cpp b/source/Files.cpp
index f5dec211..13f997e5 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -101,8 +101,8 @@ void Files::Init(const char * const *argv)
                // Special case, for Linux: the resource files are not in the same place as
                // the executable, but are under the same prefix (/app or /app/local).
-               static const filesystem::path LOCAL_PATH = "/usr/local/";
-               static const filesystem::path STANDARD_PATH = "/usr/";
-               static const filesystem::path RESOURCE_PATH = "share/games/endless-sky/";
+               static const filesystem::path LOCAL_PATH = "/app/local/";
+               static const filesystem::path STANDARD_PATH = "/app/";
+               static const filesystem::path RESOURCE_PATH = "/share/games/endless-sky/";

                const auto IsParent = [](const auto parent, const auto child) -> bool {
                        if(distance(child.begin(), child.end()) < distance(parent.begin(), parent.end()))
