diff --git a/source/Files.cpp b/source/Files.cpp
index ee0af1371..4e7d6b94f 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -101,7 +101,7 @@ void Files::Init(const char * const *argv)
 		// Special case, for Linux: the resource files are not in the same place as
 		// the executable, but are under the same prefix (/usr or /usr/local).
 		static const filesystem::path LOCAL_PATH = "/usr/local/";
-		static const filesystem::path STANDARD_PATH = "/usr/";
+		static const filesystem::path STANDARD_PATH = "/app/";
 		static const filesystem::path RESOURCE_PATH = "share/games/endless-sky/";
 
 		const auto IsParent = [](const auto parent, const auto child) -> bool {
