diff --git a/source/GameWindow.cpp b/source/GameWindow.cpp
index c6b1c1d6..e03c5c04 100644
--- a/source/GameWindow.cpp
+++ b/source/GameWindow.cpp
@@ -161,7 +161,13 @@ bool GameWindow::Init()
        // Initialize GLEW.
 #ifndef __APPLE__
        glewExperimental = GL_TRUE;
-       if(glewInit() != GLEW_OK){
+       GLenum err = glewInit();
+#ifdef GLEW_ERROR_NO_GLX_DISPLAY
+       if(err != GLEW_OK && err != GLEW_ERROR_NO_GLX_DISPLAY)
+#else
+       if(err != GLEW_OK)
+#endif
+       {
                ExitWithError("Unable to initialize GLEW!");
                return false;
        }
