diff --git a/source/Files.cpp b/source/Files.cpp
index f5dec21..8547efd 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -100,8 +100,8 @@ void Files::Init(const char * const *argv)
#if defined __linux__ || defined __FreeBSD__ || defined __DragonFly__
		// Special case, for Linux: the resource files are not in the same place as
		// the executable, but are under the same prefix (/usr or /usr/local).
-		static const filesystem::path LOCAL_PATH = "/usr/local/";
-		static const filesystem::path STANDARD_PATH = "/usr/";
+		static const filesystem::path LOCAL_PATH = "/usr/local";
+		static const filesystem::path STANDARD_PATH = "/usr";
		static const filesystem::path RESOURCE_PATH = "share/games/endless-sky/";

		const auto IsParent = [](const auto parent, const auto child) -> bool {
