diff --git a/source/Files.cpp b/source/Files.cpp
index bb03ecfc0..c888703e7 100644
--- a/source/Files.cpp
+++ b/source/Files.cpp
@@ -123,17 +123,20 @@ void Files::Init(const char * const *argv)
 
 #if defined __linux__ || defined __FreeBSD__ || defined __DragonFly__
 		// Special case, for Linux: the resource files are not in the same place as
-		// the executable, but are under the same prefix (/usr or /usr/local).
+		// the executable, but are under the same prefix (/usr or /usr/local or /app).
 		// When used as an iterator, a trailing / will create an empty item at
 		// the end, so parent paths do not include it.
 		static const filesystem::path LOCAL_PATH = "/usr/local";
 		static const filesystem::path STANDARD_PATH = "/usr";
+		static const filesystem::path FLATHUB_PATH = "/app";
 		static const filesystem::path RESOURCE_PATH = "share/games/endless-sky/";
 
 		if(IsParent(LOCAL_PATH, resources))
 			resources = LOCAL_PATH / RESOURCE_PATH;
 		else if(IsParent(STANDARD_PATH, resources))
 			resources = STANDARD_PATH / RESOURCE_PATH;
+		else if(IsParent(FLATHUB_PATH, resources))
+			resources = FLATHUB_PATH / RESOURCE_PATH;
 #endif
	}
	// If the resources are not here, search in the directories containing this
